src/createTimestamps.ts

```diff
@@ -196,30 +195,27 @@ function createOriginalStartDurations (
 
   cues.forEach((cue) => {
     const originalCue = runner?.originalCues[cue.id]
+    const daysToAdd = cue.startDatePlus || 0
+    const adjustedStartTime = cue.startTime && applyDate(cue.startTime, addDays(firstDay, daysToAdd), { timezone })
```

Comment:
This may sound a bit pedantic, but you should also pass `{ timezone }` to `addDays`
```suggestion
    const adjustedStartTime = cue.startTime && applyDate(cue.startTime, addDays(firstDay, daysToAdd, { timezone }), { timezone })
```